lib/new_relic/control/frameworks/rails6.rb

```diff
@@ -0,0 +1,14 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/rpm/blob/master/LICENSE for complete details.
+
+require 'new_relic/control/frameworks/rails4'
+
+module NewRelic
+  class Control
+    module Frameworks
+      class Rails6 < NewRelic::Control::Frameworks::Rails4
```

Comment:
Should this perhaps inherit from Rails5?

Reply:
Definitely, apologies on this. It should be I've copy pasta'd wrong.